spanner/google/cloud/spanner_v1/session.py

```diff
@@ -308,7 +311,7 @@ def run_in_transaction(self, func, *args, **kw):
                 txn.commit()
             except Aborted as exc:
                 del self._transaction
-                _delay_until_retry(exc, deadline)
+                _delay_until_retry(exc, deadline, attempts)
```

Comment:
Is it an error here?
`try-except` then delay before retry and what next? I assume `continue` statement must be here, like in lines 300-302, am I right?

Reply:
I don't think continue is strictly needed here. line 300-302 needs to avoid executing the commit. In the second case I don't think that is a concern.